user_docs/en/userGuide.md

```diff
@@ -2387,6 +2389,37 @@ By default only three modes are included.
 Note that it is necessary to check at least one mode.
 This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.
 
+##### Volume of other applications {#OtherAppVolume}
+
+This slider allows you to adjust the volume of all currently running applications other than NVDA.
+This volume setting will apply to all other applications sound output, even if they start after this setting is changed.
+This volume can also be controlled via the following keyboard commands from anywhere:
+
+| Name | Key | Description |
+|---|---|---|
+| Increase applications volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
+| Decrease applications volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |
+
+This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.
+
+##### Applications volume adjuster status {#AppsVolumeAdjusterStatus}
```

Comment:
@CyrilleB79 I see there is a sentence at the top now, is that updated since this comment?

Also re modes, I see mention of modes in the code, but not (now?) in the User Guide.  While the terminology should ideally be consistent, the presentation to the user in this case should be - I see the previous notes on Sound Split talk about modes of THAT function.

Reply:
> @CyrilleB79 I see there is a sentence at the top now, is that updated since this comment?

Yes, @mltony has added this sentence after my comment.

My opinion:
The information in this paragraph is not very clear, but that's not a matter of documentation. That's the best we can do with the current UX design. Only an alternative design such as the proposal in https://github.com/nvaccess/nvda/pull/16591#issuecomment-2255582948 could clarify more.

If you do not accept https://github.com/nvaccess/nvda/pull/16591#issuecomment-2255582948 the documentation is OK.

> Also re modes, I see mention of modes in the code, but not (now?) in the User Guide. While the terminology should ideally be consistent, the presentation to the user in this case should be - I see the previous notes on Sound Split talk about modes of THAT function.

I do not understand what you mean. It's worth noting that "mode" was appearing either in the UG or in the GUI for the newly introduced features; but @mltony has modified the wording to answer my comment on this topic.

Regarding the existing features, e.g. sound split, changing the wording is off-topic of this PR in any case. And I wonder if it may even be desirable at all considering that people are already used to the current wording.